cypress/integration/ui_spec.js

```diff
@@ -247,6 +247,81 @@ describe("UI", () => {
 				.find("[class^=error__]")
 				.should("not.exist");
 		});
+
+		it("should re-enable the input field after sending the message is successful", () => {
+			const testMessage = `Test message ${Date.now()}`;
+
+			visitHome();
+			clickOnLauncher();
+			sendMessage(testMessage);
+			findMessage(testMessage);
+
+			// Validate that api error is visible
+			cy.get("@app")
+				.find("[class^=chat__]")
+				.should("be.visible")
+				.find("[class^=footer__]")
+				.should("be.visible")
+				.find("[class^=text__]")
+				.should("be.visible")
+				.find("textarea")
+				.should("be.visible")
+				.should("be.enabled");
+		});
+
+		it("should re-enable the input field after sending the message is unsuccessful", () => {
+			const testMessage = `Test message ${Date.now()}`;
+
+			// Make sure that posting the message fails
+			cy.intercept("POST", "*/**/messages", {
+				statusCode: 400,
+				body: {
+					status: "ERROR",
+					notifications: [
+						{
+							type: "error",
+							message: "Some specific error",
+						},
+					],
+				},
+			});
+
+			visitHome();
+			clickOnLauncher();
+			sendMessage(testMessage);
+
+			// Validate that api error is visible
+			cy.get("@app")
+				.find("[class^=chat__]")
+				.should("be.visible")
+				.find("[class^=footer__]")
+				.should("be.visible")
+				.find("[class^=text__]")
+				.should("be.visible")
+				.find("textarea")
+				.should("be.visible")
+				.should("be.enabled");
+		});
+
+		it("should re-enable the input field after trying to send an empty message", () => {
```

Comment:
Hij re-enabled de input field niet, hij disabled hem niet, dus zou de message iets van "input field should still be enabled.."

Reply:
In de code doen we hem wel degelijk disablen als je een bericht stuurt en weer enablen als je bericht verstuurd is

https://github.com/parley-messaging/web-library/blob/feature/fixEmptyMessages/src/UI/ReplyActions.jsx#L20-L39

Reply:
Dat klopt, maar in het geval van een leeg bericht komt hij niet op die 2 plekken. Daarom is het punt van gydo dat hij nu nooit wordt disabled bij een leeg bericht en dus ook niet wordt **re-enabled**. Hij struikelt dus over de naamgeving van de test. Ik verwacht dat een `'it should not disable the input field when you try to send an empty message'`

Reply:
Ah ik had hem inderdaad verkeerd begrepen. Wat ik test nu is alleen of die `enabled` is ongeacht wat de status ervoor was. 

In m'n nieuwe commit heb ik er voor gezorgd dat we nu daadwerkelijk de `enabled` testen **tijdens** dat de request loopt.

Reply:
@daanleenders kan jij een nieuwe review doen en deze conv resolven als die ok is?

Reply:
Dit is nu akkoord. Had alleen nog een leesbaarheids issue met 1 van de nieuwe tests die a.d.v. deze conversatie is toegevoegd